lib/chef/provider/cron.rb

```diff
@@ -220,9 +220,14 @@ def get_crontab_entry
         [ :mailto, :path, :shell, :home ].each do |v|
           newcron << "#{v.to_s.upcase}=\"#{new_resource.send(v)}\"\n" if new_resource.send(v)
         end
+        idempotency = true
```

Comment:
This idempotency is not required if you know that the resource is not idempotent if we are using the same environment variable which is declared in properties.